packages/runtime/src/Runtime.ts

```diff
@@ -191,6 +191,14 @@ export abstract class Runtime<TConfig extends RuntimeConfig = RuntimeConfig> {
         this._anonymousServices = Container.get<AnonymousServices>(AnonymousServices);
     }
 
+    public async validateUsedBackboneVersion(): Promise<Result<void>> {
```

Comment:
"used" backbone version sounds quite off. WTB `checkBackboneCompatiblity`? This naming has to be aligned everywhere then.

Reply:
Both are fine to me